packages/react-app/src/pages/api/customers/user.ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
controller for GET customers/user (combines discord guilds and bankless customers)